src/main/java/com/smartling/connector/hubspot/sdk/rest/api/EmailsRawApi.java

```diff
@@ -12,4 +12,8 @@
     @RequestLine("PUT /marketing-emails/v1/emails/{email_id}")
     @Headers("Content-Type: application/json")
     String update(@Param("email_id") String emailId, String emailAsJson);
+
+    @RequestLine("PUT /cosemail/v1/emails/{email_id}")
```

Comment:
I propose adding a comment that will be useful in the future when we need to update the SDK.
```suggestion
    // This is a workaround specifically made for updating A/B Emails. The standard endpoint throws an error
    // "Changing the abTestId is not allowed."
    @RequestLine("PUT /cosemail/v1/emails/{email_id}")
```

Reply:
Done